salt/metalk8s/orchestrate/solutions/files/operator/deployment.yaml.j2

```
@@ -49,7 +49,18 @@ spec:
           volumeMounts:
             - name: operator-config
               mountPath: /etc/config
+{%- if webhook_enabled is defined and webhook_enabled %}
+            - name: cert
+              mountPath: /cert
+              readOnly: true
+{%- endif %}
       volumes:
         - name: operator-config
           configMap:
             name: {{ solution }}-operator-config
+{%- if webhook_enabled is defined and webhook_enabled %}
+        - name: cert
+          secret:
+            defaultMode: 420
+            secretName: {{ solution }}-cert
+{%- endif %}
```

nitpick the `is defined` is not needed here since it will always be defined but fine
```suggestion
{%- if webhook_enabled %}
            - name: cert
              mountPath: /cert
              readOnly: true
{%- endif %}
      volumes:
        - name: operator-config
          configMap:
            name: {{ solution }}-operator-config
{%- if webhook_enabled %}
        - name: cert
          secret:
            defaultMode: 420
            secretName: {{ solution }}-cert
{%- endif %}
```

---

as in the unit tests `formulas`, this variable is not defined. So I add this check to cover the case.